app/assets/javascripts/live-search.js

```diff
@@ -40,6 +40,7 @@
       }
     },
     pageTrack: function(){
+      ga("set", "page", window.location.pathname + window.location.search);
       GOVUK.analytics.setResultCountDimension(liveSearch.cache().result_count);
       GOVUK.analytics.trackPageview();
```

Comment:
You should just be able to update this to `GOVUK.analytics.trackPageview(window.location.pathname + window.location.search);` rather than calling `ga` directly.